js/condition.funnelenvy.js

```diff
@@ -27,21 +27,26 @@
   Drupal.smartContent.plugin.Field['funnelenvy'] = function (condition) {
     let key = condition.field.pluginId.split(':')[1];
         Drupal.smartContent.funnelenvy = new Promise((resolve, reject) => {
-          Promise.resolve(Drupal.smartContent.waitForFunnelenvy).then(function(isFeAvailable){
-            if(isFeAvailable){
+          Promise.resolve(Drupal.smartContent.waitForFunnelenvy).then(function(Funnelenvy){
```

Comment:
lets use the appropriate JS casing `funnelEnvy` here

Reply:
@axs89 Fixed